wp-content/themes/reconasia/assets/_scss/pages/archive.scss

```diff
@@ -82,3 +82,39 @@
     display: none;
   }
 }
+
+.cards__container {
+  @include vw100();
+
+  @include breakpoint('large') {
+    display: flex;
+  }
+}
+
+.card {
```

Comment:
Note that the card is meant to have a box-shadow, of the medium size, applied. There's a note about it on [Figma](https://www.figma.com/file/kIHZmR756odImUSgXxPnp0/RA-Handoff?node-id=63%3A10184).